README.md

```diff
@@ -53,6 +53,36 @@ python metriq_gym/run.py poll --job_id <METRIQ_GYM_JOB_ID>
 
 where `<METRIQ_GYM_JOB_ID>` is the assigned job ID of the job that was dispatched as provided by `metriq-gym`. 
 
+### Listing jobs
+
+You can view all the jobs that have been dispatched by using the `--list-jobs` action. This will read the `jobs_file`
```

Comment:
I am tempted to get rid of letting the user pick`jobs_file` as argument. We don't want to give too many options to the user. It's an internal file anyway. Wdyt?

Reply:
I agree. This kind of detail should be hidden from the user. I've got ahead and made the necessary updates to impose the jobs file as an attribute in the `JobManager` class in  71936ee.